google_symptoms/delphi_google_symptoms/run.py

```diff
@@ -94,15 +94,18 @@ def run_module(params):
 
         if len(df_pull) == 0:
             continue
-        for metric, smoother in product(
-                METRICS+[COMBINED_METRIC], SMOOTHERS):
+        for metric, smoother in product(COMBINED_METRIC, SMOOTHERS):
             logger.info("generating signal and exporting to CSV",
                         geo_res=geo_res,
                         metric=metric,
                         smoother=smoother)
-            df = df_pull.set_index(["timestamp", "geo_id"])
-            df["val"] = df[metric].groupby(level=1
-                                           ).transform(SMOOTHERS_MAP[smoother][0])
+            df = df_pull#.set_index(["timestamp", "geo_id"])
+            df["val"] = df[metric].astype(float)
+            df["val"] = df[["geo_id", "val"]].groupby(
+                "geo_id")["val"].transform(
+                SMOOTHERS_MAP[smoother][0].smooth)
+            #df["val"] = df[metric].groupby(level=1
+            #                               ).transform(SMOOTHERS_MAP[smoother][0].smooth)
```

Comment:
Please remove out-of-date code.

```suggestion
            df = df_pull
            df["val"] = df[metric].astype(float)
            df["val"] = df[["geo_id", "val"]].groupby(
                "geo_id")["val"].transform(
                SMOOTHERS_MAP[smoother][0].smooth)
```